tests/wpt/web-platform-tests/css/cssom/stylesheet-replacedata-dynamic-ref.html

```diff
@@ -0,0 +1,6 @@
+<!doctype html>
+<title>(Ref #1) Stylesheet replaceData Dynamic</title>
```

Comment:
Ditto re author, and usually the title for the reference is just `CSS Test Reference`.

Reply:
Got it!